src/content/it/2020/resource-hints.md

```diff
@@ -0,0 +1,461 @@
+---
+part_number: IV
+chapter_number: 21
+title: Resource Hints
+description: Resource Hints chapter of the 2020 Web Almanac covering usage of dns-prefetch, preconnect, preload, prefetch, Priority Hints, and native lazy loading.
+authors: [Zizzamia]
+reviewers: [jessnicolet, pmeenan, giopunt, mgechev, notwillk]
+analysts: [khempenius]
+translators: []
+Zizzamia_bio: Leonardo is a Staff Software Engineer at <a href="https://www.coinbase.com/">Coinbase</a>, leading web performance and growth initiatives. He curates the <a href="https://ngrome.io">NGRome Conference</a>. Leo also maintains the <a href="https://github.com/Zizzamia/perfume.js">Perfume.js</a> library, which helps companies prioritize roadmaps and make better business decisions through performance analytics.
+discuss: 2057
+results: https://docs.google.com/spreadsheets/d/1lXjd8ogB7kYfG09eUdGYXUlrMjs4mq1Z7nNldQnvkVA/
+queries: 21_Resource_Hints
+featured_quote: During the past year resource hints increased in adoption, and they have become essential APIs for developers to have more granular control over many aspects of resource prioritizations and ultimately, user experience.
+featured_stat_1: 33%
+featured_stat_label_1: Sites using <code>dns-prefetch</code>
+featured_stat_2: 9%
+featured_stat_label_2: Sites using <code>preload</code>
+featured_stat_3: 4.02%
+featured_stat_label_3: Sites using native lazy loading
+---
+
+## Introduzione
+
+<!-- Over the past decade [resource hints](https://www.w3.org/TR/resource-hints/) have become essential primitives that allow developers to improve page performance and therefore the user experience. -->
+
+Negli ultimi dieci anni [resource hints](https://www.w3.org/TR/resource-hints/) sono diventati primitivi essenziali che consentono agli sviluppatori di migliorare le prestazioni della pagina e quindi la user experience.
```

Comment:
@chefleo we might want to use a different word for `essential primitives`, dire primitive essenziali suona strano. Also instead of saying `della pagina e quindi la user experience.` suona meglio `della pagina e l'user experience.`